lib/lambdalocal.js

```diff
@@ -7,6 +7,7 @@
  */
 
 var logger = require('winston');
+var dotenv = require('dotenv');
```

Comment:
We got too much depedencies, can you add a system to auto-detect (catching an exception) in order to disable the feature if `dotenv` is not installed ?

BTW, if you could do the same thing with `winston` it would be great